.teamcity/src/main/kotlin/configurations/SmokeTests.kt

```diff
@@ -8,7 +8,7 @@ import model.CIBuildModel
 import model.Stage
 
 class SmokeTests(model: CIBuildModel, stage: Stage, testJava: JvmCategory, task: String = "smokeTest", splitNumber: Int = 1) : BaseGradleBuildType(stage = stage, init = {
-    id("${model.projectId}_${task.toCapitalized()}s${testJava.version.name.toCapitalized()}")
```

Comment:
❓ Aren't we running smoke tests on different Java versions? If yes, then maybe we could do something like another counter again instead of removing the Java version completely from the name.